cmake_files/dependencies.cmake

```diff
@@ -44,80 +89,41 @@ if(openmp AND OPENMP_FOUND)
   set_target_properties(openmp::openmp PROPERTIES
     INTERFACE_COMPILE_OPTIONS "${OpenMP_CXX_FLAGS}"
     INTERFACE_LINK_LIBRARIES  "${OpenMP_CXX_FLAGS}")
-
-  find_package(FFTW3 REQUIRED DOUBLE SERIAL COMPONENTS OPENMP)
-  set(FFTW3_DOUBLE_LIBRARY fftw3::double::serial)
-  if(TARGET fftw3::double::openmp)
-    list(APPEND FFTW3_DOUBLE_LIBRARY fftw3::double::openmp)
-    set(PURIFY_OPENMP_FFTW TRUE)
-  endif()
 else()
   # Set to FALSE when OpenMP is not found or not requested
   set(PURIFY_OPENMP FALSE)
-  find_package(FFTW3 REQUIRED DOUBLE)
-  set(FFTW3_DOUBLE_LIBRARY fftw3::double::serial)
 endif()
 
+find_package(FFTW3)
```

Comment:
This was a conditional requirement when we had regressions=on, but we removed the regressions all together. Does anything else need FFTW3?

Reply:
It looks like it's always used in [operators.h](https://github.com/astro-informatics/purify/blob/development/cpp/purify/operators.h)